src/core/muselementfactory.cpp

```diff
@@ -164,8 +164,8 @@ bool CAMusElementFactory::configureClef(CAStaff* staff,
 {
     bool success = false;
     if (staff && staff->voiceList().size()) {
-        mpoMusElement = new CAClef(_eClef, staff, 0, _iClefOffset);
-        success = staff->voiceList()[0]->insert(right, mpoMusElement);
+        mpoMusElement = std::make_shared<CAClef>(_eClef, staff, 0, _iClefOffset);
+        success = staff->voiceList()[0]->insert(right, mpoMusElement.get());
```

Comment:
What happens to QList<CAMusElement*>? Should it become QList<std::shared_ptr<CAMusElement>>? If so, we need to update swig helpers which transform QList<CAMusElement*> type to native python lists.